eslint.config.mjs

```diff
@@ -0,0 +1,18 @@
+import simpleImportSort from 'eslint-plugin-simple-import-sort'
+
+export default [
+  {
+    files: ['**/*.mjs', '**/*.js'],
+    plugins: {
+      'simple-import-sort': simpleImportSort,
+    },
+    rules: {
+      'simple-import-sort/exports': 'error',
```

Comment:
it should also use prettier so i would expect to see "prettier/prettier": "error" and prettier config and plugin